vite.config.ts

```diff
@@ -0,0 +1,30 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import viteTsconfigPaths from 'vite-tsconfig-paths';
+import svgrPlugin from 'vite-plugin-svgr';
+import EnvironmentPlugin from 'vite-plugin-environment';
```

Comment:
1. Can this file be placed in a `.config` or `config` directory like we have done with the API?
2. It'll reduce the clutter in the root directory